main.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"database/sql"
+	"fmt"
+	"github.com/joho/godotenv"
+	"go1f/pkg/api"
+	"go1f/pkg/db"
+	"go1f/pkg/server"
+	"os"
+)
+
+func main() {
+
+	_ = godotenv.Load()
+	TODO_PASSWORD := os.Getenv("TODO_PASSWORD")
+	api.PASS = TODO_PASSWORD
+
+	TODO_DBFILE := os.Getenv("TODO_DBFILE")
```

Comment:
Локальные переменные называют с маленькой буквы и в `pascalCase`
```suggestion
	todoPassword := os.Getenv("TODO_DBFILE")
```